test/feature/test_mkd.py

```diff
@@ -0,0 +1,398 @@
+import pytest
+import kornia.testing as utils  # test utils
+
+from torch.testing import assert_allclose
+from torch.autograd import gradcheck
+from kornia.feature.mkd import *
+
+
+@pytest.mark.parametrize("ps", [5, 13, 25])
+def test_get_grid_dict(ps):
+    grid_dict = get_grid_dict(ps)
+    param_keys = ['x', 'y', 'phi', 'rho']
+    assert set(grid_dict.keys()) == set(param_keys)
+    for k in param_keys:
+        assert grid_dict[k].shape == (ps, ps)
+
+
+@pytest.mark.parametrize("d1,d2",
+                         [(1, 1), (1, 2), (2, 1), (5, 6)])
+def test_get_kron_order(d1, d2):
+    out = get_kron_order(d1, d2)
+    assert out.shape == (d1 * d2, 2)
+
+
+class TestMKDGradients:
```

Comment:
tests are great! you missing `dtype` in each test. BTW, have you tried to run in GPU ?

Reply:
optional but would really help to implement as a class. Check: https://github.com/kornia/kornia/blob/master/test/enhance/test_normalize.py#L223

Reply:
can you also try test with JIT ?

Reply:
GPU - i can see that there is a conftest.py file, that adds device as an argument, so i think it is running GPU. 
I am not familiar with JIT, so haven't coded any tests for it.

Reply:
check this: https://github.com/kornia/kornia/blob/master/test/enhance/test_normalize.py#L273
if everything it's well done, should be plug and play.